### ISSUE TITLE
Lambda output_shape

### ISSUE THREAD

**Issue by hzfei** [keras-team/keras#21089]
tensorflow_version=2.19
code:
features_expand_dims = Lambda(lambda x:K.expand_dims(x,axis=-1))(features)

err:
We could not automatically infer the shape of the Lambda's output. Please specify the `output_shape` argument for this Lambda layer.

**Comment by dhantule** [comment]
Hi @hzfei, thanks for reporting this. I am unable to reproduce.  Can you provide some standalone reproducible code ?

**Comment by hzfei** [comment]
"Oops, I just updated the TF version. My code was written using keras==2.15, but the TF version is now 2.19. You're right, it should be from keras import backend as K → from keras import ops as K."

**Comment by hzfei** [comment]
sorry, I just updated the TF version. My code was written using keras==2.15, but the keras version is now 3.9. You're right, it should be ”from keras import backend as K → from keras import ops as K“.

**Comment by dhantule** [comment]
Hi @hzfei, can we close the issue if the error's resolved ?
